vscode_extension/package.json

```diff
@@ -0,0 +1,58 @@
+{
+    "name": "AaC",
+    "displayName": "Architecture As Code",
+    "description": "TODO: Fill me in!",
+    "version": "0.0.1",
+    "engines": {
+        "vscode": "^1.60.0"
+    },
+    "categories": [
+        "Other"
+    ],
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/jondavid-black/AaC.git"
+    },
+    "activationEvents": [
+        "onStartupFinished"
+    ],
+    "main": "./out/extension.js",
+    "contributes": {
+        "taskDefinitions": [
+            {
+                "type": "aac",
+                "properties": {
+                    "task": {
+                        "type": "string",
+                        "description": "AaC Command"
+                    }
+                },
+                "required": []
+            }
+        ]
+    },
+    "scripts": {
+        "vscode:prepublish": "yarn run compile",
+        "compile": "tsc -p ./",
+        "watch": "tsc -watch -p ./",
+        "pretest": "yarn run compile && yarn run lint",
+        "lint": "eslint src --ext ts",
+        "test": "node ./out/test/runTest.js",
+        "package": "vsce package --yarn -o aac.vsix"
+    },
+    "devDependencies": {
+        "@types/glob": "^7.2.0",
+        "@types/mocha": "^9.0.0",
+        "@types/node": "14.x",
+        "@types/vscode": "^1.60.0",
+        "@typescript-eslint/eslint-plugin": "^5.9.1",
+        "@typescript-eslint/parser": "^5.9.1",
+        "@vscode/test-electron": "^2.0.3",
+        "eslint": "^8.6.0",
+        "glob": "^7.2.0",
+        "mocha": "^9.1.3",
+        "typescript": "^4.5.4",
+        "vsce": "^2.6.4"
+    },
+    "dependencies": {}
```

Comment:
Are there really no dependencies? This seems kind of unlikely - were some of the dependencies added to the `devDependencies` block (e.g. `@types/vscode`, `vsce`, etc seem like they should probably be general dependencies)?

Reply:
Correct, the plugin itself doesn't currently use any dependencies that aren't already provided to it via the host vscode environment. If we were to implement diagraming libraries for visualization of models, then we would need to populate the dependencies block with that.